### ISSUE TITLE
Details about the development set

### ISSUE THREAD

**Issue by hitercs** [HazyResearch/bootleg#60]
Hi,

Thanks for your great work!
I want to conduct some analysis for Bootleg using the Wikipedia development set you used in the paper. 
I downloaded the wiki data using [this script](https://github.com/HazyResearch/bootleg/blob/master/tutorials/download_wiki.sh). However, I didn't find a readme to explain each file's usage. It would be great if you can explain a bit more about them. 

Specifically, in the paper, I saw there are a total 5.7M sentences. in Section B.1. 
In the downloaded data: 
train.jsonl consists of 51056341 sentences, dev.jsonl  consists of 4859374 sentences, test.jsonl consists of 4880947 sentences. 
The total number of sentences is 60,796,662. Is this data updated? 

Do train.jsonl, dev.jsonl and test.jsonl follow 80 / 10 / 10 split with respect to the number of mentions?

Also what is the meaning of merged.jsonl and merged_sample.jsonl? 

Also, I saw there are three models of Bootleg, two on full Wikipedia data (one for ablation and one for the benchmarks) and one on micro data. The released model is the ablation model trained with 80 / 10 / 10 partitions (not the 96 / 2 / 2 dev split)?

What does the various slices mean in the jsonl data. I saw the [documentation](https://bootleg.readthedocs.io/en/latest/advanced/distributed_training.html#evaluating-with-slices) explains about the 5 unif_* slices, how about the remaining ones, what does they mean?

Thanks very much for your help and time!

**Comment by lorr1** [comment]
Hey Shuang! Sorry it took a few days. Happy to explain more. So we did a second release of a new model _after_ the paper came out. We just released the four models (uncased vs cased and small vs regular). These were trained on a newer Wikipedia dump (Oct 2020, I believe) on the 80 / 10 / 10 split data that's on the AWS bucket.

During training, we don't actually do any parameter tuning using our dev sets. So merged and merged_sample is the union of our dev/test sets together. The sampled version is a subset we used to do online eval during training.

For our slices, I have three ways of naming them. Two uppercase letters at the end signify the population based on frequency.
HD = Head
TO = Torso
TL = Tail
TS = Toes
ALL = Everything

The uppercase letters in the middle filter based on candidate lists
NS = Non Signal (i.e., the mention has more than one candidate)
NPOP = Non Pop (i.e., the gold QID isn't the most popular candidate)
[blank] = all

The prefix describes the slice itself. I'll add those I remember but feel free to ask about more
wd_aff = wikidata type keyword signal pattern (1 vs 3 signifies the number of types I used to look for textual keyword patterns)
rel_aff = relation keyword signal pattern
kg_rel = is there a KG connection between this QID and another
cons = type consistency (are the sequences of the same type) (1 vs 3 is again the number of types I used)
noTnoR = no type or relation given for the gold QID

I have a private repo with all our data prep code in it that goes from raw wikipedia data to data for training. If you'd like to be added to that, let me know. Happy to share it.
